web/src/features/moreCast2/components/MoreCast2Column.tsx

```diff
@@ -152,28 +155,86 @@ export class IndeterminateField implements ColDefGenerator, ForecastColDefGenera
   }
   public generateForecastColDef = (columnClickHandlerProps: ColumnClickHandlerProps, headerName?: string) => {
     return {
-      ...this.colDefBuilder.generateForecastColDef(columnClickHandlerProps, headerName ?? this.headerName)
+      ...this.colDefBuilder.generateForecastColDef(
+        columnClickHandlerProps,
+        headerName ?? this.headerName,
+        this.preProcessEditCellProps
+      )
     }
   }
 
   public generateForecastSummaryColDef = (columnClickHandlerProps: ColumnClickHandlerProps) => {
-    return this.colDefBuilder.generateForecastColDef(columnClickHandlerProps)
+    return this.colDefBuilder.generateForecastColDef(columnClickHandlerProps, undefined, this.preProcessEditCellProps)
   }
 
   public generateColDef = () => {
-    return this.colDefBuilder.generateColDefWith(this.field, this.headerName, this.precision)
+    return this.colDefBuilder.generateColDefWith(
+      this.field,
+      this.headerName,
+      this.precision,
+      undefined,
+      this.preProcessEditCellProps
+    )
   }
 
   public generateColDefs = (columnClickHandlerProps: ColumnClickHandlerProps, headerName?: string) => {
-    return this.colDefBuilder.generateColDefs(columnClickHandlerProps, headerName ?? this.headerName, this.includeBias)
+    return this.colDefBuilder.generateColDefs(
+      columnClickHandlerProps,
+      headerName ?? this.headerName,
+      this.includeBias,
+      this.preProcessEditCellProps
+    )
   }
 }
 
-export const tempForecastField = new IndeterminateField('temp', TEMP_HEADER, 'number', 0, true)
-export const rhForecastField = new IndeterminateField('rh', RH_HEADER, 'number', 0, true)
-export const windDirForecastField = new IndeterminateField('windDirection', WIND_DIR_HEADER, 'number', 0, true)
+export const tempForecastField = new IndeterminateField(
+  'temp',
+  TEMP_HEADER,
+  'number',
+  0,
+  true,
+  (params: GridPreProcessEditCellProps) => {
+    const hasError =
+      params.props.value < -60 || params.props.value > 60 ? 'Temp must be between -60 and 60 degrees C' : false
```

Comment:
So `hasError`/`error` is interpreted as truthy when assigned a string instead of a boolean and that string gets displayed in the tooltip?

Reply:
Haha yes, hacky, but I'm going to refactor this whole approach I think because setting error values through `preProcessEditCellProps` doesn't persist for cells in view mode so errors are only showing up while editing.

Reply:
I was just about to drop a comment on that!